src/main/java/com/dpgil/pathlinker/path_linker/internal/PathLinkerResultPanel.java

```diff
@@ -0,0 +1,377 @@
+	class DiscardButtonListener implements ActionListener {
+
+		// Discard the entire currently selected result panel tab if user chooses yes
+		@Override
+		public void actionPerformed(ActionEvent e) {
+
+			int choice = JOptionPane.showConfirmDialog(null, "Discarded result will be remove permanently. Continue?");
+			if (choice != 0) return; // quit if they say no or cancel
+
+			Container btnParent = _discardBtn.getParent();
+			Container panelParent = btnParent.getParent();
+			panelParent.remove(btnParent);
+		}
+	}
+
+	/**
+	 * Listener for the paths in result table
+	 * When user selects a path in the table, set the related nodes and edges to be selected in the network view
+	 * Multiple nodes can be select at once
+	 */
+	class ResultTableListener implements ListSelectionListener {
+		@Override
+		public void valueChanged(ListSelectionEvent e) {
+			ListSelectionModel lsm = (ListSelectionModel)e.getSource();
+
+			List<CyNode> selectedNodes = CyTableUtil.getNodesInState(_originalNetwork, "selected", true);
+			List<CyEdge> selectedEdges = CyTableUtil.getEdgesInState(_originalNetwork, "selected", true);
+			
+			// clear the original selected nodes and edges from the view
+			for (CyNode node : selectedNodes) 
+				_originalNetwork.getRow(node).set(CyNetwork.SELECTED, false);
+			for (CyEdge edge : selectedEdges) 
+				_originalNetwork.getRow(edge).set(CyNetwork.SELECTED, false);
+
+			// return if nothing is selected
+			if (lsm.isSelectionEmpty()) return;
+
+			// find the index of first and last nodes
+			int minIndex = lsm.getMinSelectionIndex();
+			int maxIndex = lsm.getMaxSelectionIndex();
+
+			// check the nodes in between the indexes
+			for (int i = minIndex; i <= maxIndex; i++) {
+				if (lsm.isSelectedIndex(i)) {
+					Path currPath = _results.get(i);
+					// excluding supersource and supertarget
+					for (int j = 1; j < currPath.size() - 2; j++) {
+						CyNode node1 = currPath.get(j);
+						CyNode node2 = currPath.get(j + 1);
+
+						_originalNetwork.getRow(node1).set(CyNetwork.SELECTED, true);
+						_originalNetwork.getRow(node2).set(CyNetwork.SELECTED, true);
+
+						// add all of the directed edges from node1 to node2
+						List<CyEdge> edges = _originalNetwork.getConnectingEdgeList(node1, node2, CyEdge.Type.DIRECTED);
+						for (CyEdge edge : edges) {
+							if (edge.getSource().equals(node1) && edge.getTarget().equals(node2)) { // verifies the edges direction
+								_originalNetwork.getRow(edge).set(CyNetwork.SELECTED, true);
+							}
+						}
+						// also add all of the undirected edges from node1 to node2
+						edges = _originalNetwork.getConnectingEdgeList(node1, node2, CyEdge.Type.UNDIRECTED);
+						for (CyEdge edge : edges) _originalNetwork.getRow(edge).set(CyNetwork.SELECTED, true);
+					}
+				}
+			}
+		}
+	}
+
+	/**
+	 * Sets up all the components in the panel 
+	 */
+	private void initializePanel() {
+		this.setLayout(new GridBagLayout());
+
+		setUpDownloadBtn();
```

Comment:
Can you change Download to Export to match the other export options in Cytoscape?